packages/fluent-editor/src/table/better-table.ts

```
@@ -52,6 +54,12 @@
   tableOperationMenu: any
   columnTool: TableColumnTool
   tableScrollBar: TableScrollBar
+  tableSelector: TableSelector
+  isTableSelectorVisible: boolean
+  tableSelectorWrapper: HTMLDivElement
+  tableSelector: TableSelector
```

Biome (1.9.4)

[error] Duplicate class member name "tableSelector"



(lint/suspicious/noDuplicateClassMembers)